packages/base/position.gts

```diff
@@ -0,0 +1,16 @@
+import NumberField from './number';
+import { contains, FieldDef, field, CardDef, linksTo } from './card-api';
+
+export class Position extends FieldDef {
```

Comment:
Is the Position field meant solely to express integer grid coordinates (x,y)? If so, maybe we can consider a more specific name for this field. For example if I saw a Position field I would wonder what kind of a position it is.